_docs/README-VI.md

```diff
@@ -0,0 +1,188 @@
+<div align="center">
+  <h1>
+
+  ✍️
+
+  Handmade Blog
+
+  [![build](https://img.shields.io/github/workflow/status/ParkSB/handmade-blog/Node%20CI/master?style=flat-square)](https://github.com/ParkSB/handmade-blog/actions?query=workflow%3A%22Node+CI%22) ![node](https://img.shields.io/badge/node-%3E%3D%2010.0-brightgreen?style=flat-square) [![demo](https://img.shields.io/netlify/3f01acb3-1107-470a-914f-90d100b87d85?label=demo&style=flat-square)](https://handmade-blog.netlify.com/) [![license](https://img.shields.io/github/license/ParkSB/handmade-blog?style=flat-square)](LICENSE)
+
+  </h1>
+</div>
+
+Handmade Blog là một trình nhẹ tạo blog tĩnh cho những người muốn bắt đầu một blog nhanh chóng. Nó hỗ trợ thư mục article cho một bài đăng, thư mục work cho portfolio, nổi bật code, hỗ trợ [KaTeX](https://katex.org/), footnotes và hơn thế nữa.
+
+## Xem thử: [Tại đây](https://handmade-blog.vercel.app/)
+
+![Article page preview](https://user-images.githubusercontent.com/6410412/74097056-be43d100-4b4a-11ea-806b-7bd263d7f623.png)
+
+## Bắt Đầu
+
+1. Nhấp vào nút 'Use this template' phía trên danh sách tệp để tạo một kho lưu trữ mới. Nếu bạn muốn sử dụng miền github.io, phải đặt tên cho kho lưu trữ là `{YOUR_ID} .github.io`. (ví dụ: `betty-grof.github.io`) Đừng quên bật tùy chọn 'Include all branches' .
+
+    ![Click the 'Use this template' button](https://user-images.githubusercontent.com/6410412/93741226-f524ae00-fc26-11ea-8f88-ba634d2de66b.png)
+
+    ![Name repository to id.github.io, and enable 'Include all branches' option](https://user-images.githubusercontent.com/6410412/93741223-f48c1780-fc26-11ea-9980-8911e531a29c.png)
+
+2. Nhấp vào tab 'Settings' trong kho lưu trữ của bạn và đặt nhánh nguồn cho Trang GitHub thành nhánh `gh-pages`. GitHub Pages sẽ lưu trữ trang web của bạn dựa trên nhánh `gh-pages`. Bạn sẽ có thể truy cập trang web qua `https://{YOUR_ID}.github.io/` sau vài phút.
+
+    ![Click the 'Settings' tab](https://user-images.githubusercontent.com/6410412/93750006-d11c9900-fc35-11ea-9ac1-4f92216f28f9.png)
+
+    ![Set source branch of the github pages to gh-pages branch](https://user-images.githubusercontent.com/6410412/93741218-f2c25400-fc26-11ea-9e30-eddb9a2a3b3f.png)
+
+3. Sao chép kho lưu trữ và cài đặt các gói node.
+
+    ```shell script
+    $ git clone https://github.com/{YOUR_ID}/{REPOSITORY_NAME}.git # git clone https://github.com/betty-grof/betty-grof.github.io.git
+    $ cd {REPOSITORY_NAME} # cd betty-grof.github.io
+    $ npm install
+    ```
+
+4. Sửa đổi tệp `config.json` trong thư mục `services` để đặt tiêu đề blog và phụ đề của bạn.
+
+    ```json
+    {
+      "blogTitle": "Betty Grof",
+      "blogSubtitle": "Oh My Glob",
+      "article": {
+        "tableOfContents": true 
+      }
+    }
+    ```
+
+5. Khởi động máy chủ cục bộ tại `http://localhost:1234/`. Chạy lệnh `npm start` mở máy chủ cục bộ dựa trên thư mục` server`.
+
+    ```shell script
+    $ npm start
+    ```

```

Comment:
The contents of the fourth and fifth steps have been changed. Please check if it's translated according to the latest README.md.

```
4. Customize a some texts such as title of the navigation(in `app/templates/navigations.ejs`), and run `npm run build`.

    ```html
    <nav>
      <a class="logo-link" href="/">
        <h1>CUSTOMIZED BLOG TITLE</h1>
        <span>customized blog subtitle</span>
      </a>
      <small>
        <a id="about" class="info-link" href="/about.html">👀About</a> /
        <a id="works" class="info-link" href="/works.html">🔥Works</a> /
        <a id="articles" class="info-link" href="/articles.html">📚Articles</a>
      </small>
    </nav>
    ```
    ``` shell script
    $ npm run build
    ```

5. Run `npm start` script to start a local server listening on `http://localhost:8080/`. The local server is based on `dist` directory.

    ```shell script
    $ npm run build
    $ npm start
    ```
```